db/external_sst_file_basic_test.cc

```diff
@@ -1901,27 +1901,282 @@ TEST_F(ExternalSSTFileBasicTest, AtomicReplaceData) {
     files.push_back(std::move(file4));
   }
 
+  const Snapshot* snapshot = db_->GetSnapshot();
+
   auto seqno_before_ingestion = db_->GetLatestSequenceNumber();
   ASSERT_OK(db_->IngestExternalFile(files, ifo));
   // Overlapping files each occupy one new sequence number.
   ASSERT_EQ(db_->GetLatestSequenceNumber(), seqno_before_ingestion + 3);
```

Comment:
Do you know why is it using 3 seqno instead of 2? I thought with #13064, we only have two batches of files to ingest here

Reply:
Yeah, for this part, we didn't make the change to use less seqnos when files are ingested in batches. We can do that in the future, but for now when input files overlap, each just got assigned a separate seqno.